sdks/python/apache_beam/examples/wordcount.py

```diff
@@ -117,7 +117,12 @@ def _add_argparse_args(cls, parser):
     if query_result['counters']:
       empty_lines_counter = query_result['counters'][0]
       logging.info('number of empty lines: %d', empty_lines_counter.committed)
-    # TODO(pabloem)(BEAM-1366): Add querying of MEAN metrics.
+
+    word_lengths_filter = MetricsFilter().with_name('word_len_dist')
+    query_result = result.metrics().query(word_lengths_filter)
+    if query_result['distributions']:
```

Comment:
Should it be an error if a certain type of metrics is expected but not available?